L1Trigger/RPCTrigger/src/RPCPac.cc

```diff
@@ -114,7 +114,7 @@ RPCPacMuon RPCPac::runEnergeticPatternsGroups(const RPCLogCone& cone) const {
           int strip = itLP->first;
           if (iEGroup->m_GroupShape.getLogStripState(logPlane, strip)) {
             firedPlanes = firedPlanes | one;
-            firedPlanesCount++;
+            // firedPlanesCount++;
```

Comment:
Is there a reason these are commented instead of removed?

Reply:
Hello @aloeliger,
I didn't remove it in this case because it is used below in an already commented statement (https://github.com/cms-sw/cmssw/blob/master/L1Trigger/RPCTrigger/src/RPCPac.cc#L122-L137). Normally, if the unused variables is used as part of a commented print out, we just remove the print too. In this case, since I am not aware of the original intention of the author I kept the commented code below. In any case, I can remove it if you prefer so.
Thanks!

Reply:
I have no preference in this case. I was just curious if there was something about this that merited special attention. Thanks!